backend/pkg/exporter/modules/execution_deposits_exporter.go

```diff
@@ -271,8 +313,13 @@ func (d *executionDepositsExporter) fetchDeposits(fromBlock, toBlock uint64) (de
 			return nil, fmt.Errorf("nil deposit-log")
 		}
 
-		depositLog :=
-			depositLogIterator.Event
+		depositLog := depositLogIterator.Event
+
+		if d.exportedTxsCache.Contains(fmt.Sprintf("%x", depositLog.Raw.TxHash.Bytes())) {
+			log.Debugf("skipping already exported deposit-tx: block: %d, tx: %x", depositLog.Raw.BlockNumber, depositLog.Raw.TxHash.Bytes())
+			continue
+		}
```

Comment:
this is wrong and very dangerous. i meant only caching the validity of the signature, not skipping seen txns

the current solution can result in the following issue:
- there are 3 deposit txns on chain, each with one deposit: `txnA:0`, `txnB:1`, `txnC:2` (`:n` is the merkle tree index)
  - **`txnA:0`, `txnB:1`, `txnC:2`** gets pushed to the db
- the chain gets reorged and a new txn (`txnD`) with 1 deposit get inserted after `txnA`
  - there are now 4 deposits on chain: `txnA:0`, `txnD:1`, `txnB:2`, `txnC:3`
  - the lru cache skips the re-processing of txnB & txnC
  -  **`txnD:1`** gets pushed to the db
- a new txn with a single deposit gets added (`txnE`)
  - **`txnE:4`** gets pushed to the db

the result db state would be
-  `txnA:0`, `txnD:1`, `txnC:2`, `txnE:4`

instead of 
- `txnA:0`, `txnD:1`, **`txnB:2`, `txnC:3`**, `txnE:4`

Reply:
removed cache